src/java/org/apache/cassandra/db/SSTableImporter.java

```diff
@@ -182,19 +185,35 @@ synchronized List<String> importNewSSTables(Options options)
 
         try (Refs<SSTableReader> refs = Refs.ref(newSSTables))
         {
+            abortIfDraining();
             cfs.getTracker().addSSTables(newSSTables);
             for (SSTableReader reader : newSSTables)
             {
                 if (options.invalidateCaches && cfs.isRowCacheEnabled())
                     invalidateCachesForSSTable(reader);
             }
-
+        }
+        catch (Throwable t)
+        {
+            logger.error("[{}] FaiFailed adding sstables", importID, t);
+            throw new RuntimeException("Failed adding sstables", t);
```

Comment:
Nit: `sstables` -> `SSTables`

Reply:
Fixed here and in six other places within this method.